### PR TITLE
Feat: enhance combiners sudoers

### PR DESCRIPTION
**Pull request by TZ3070**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

enhance combiners sudoers

### CODE REVIEW

**Comment by TZ3070** [comment]
@JoySnow @chenlizhong Please have a review

File: insights/combiners/sudoers.py [hunk 1]
```diff
@@ -41,8 +41,11 @@ class Sudoers(SudoersBase):
     """
     def __init__(self, sudoers):
         self.lines = []
+        self.data = {}
         first = False
         for sdr in sorted(sudoers, key=lambda x: x.file_path):
+            self.data[sdr.file_path] = []
```
Comment (chenlizhong) [inline insights/combiners/sudoers.py:47]: Can we remove this line?

Comment (TZ3070) [reply insights/combiners/sudoers.py]: @chenlizhong Updated

File: insights/combiners/sudoers.py [hunk 2]
```diff
@@ -41,8 +41,10 @@ class Sudoers(SudoersBase):
     """
     def __init__(self, sudoers):
         self.lines = []
+        self.data = {}
```
Comment (JoySnow) [inline insights/combiners/sudoers.py:44]: @TZ3070 , we better also add this new attribute in the docstrings' `Attributes` part, just as `lines(list)` .

**Review by JoySnow — CHANGES_REQUESTED** [review]

File: insights/combiners/sudoers.py (hunk 2 above)
Comment (TZ3070) [reply insights/combiners/sudoers.py:47]: @JoySnow Updated

**Review by JoySnow — APPROVED** [review]
Thanks @TZ3070 for the changes, LGMT now.

**Review by chenlizhong — APPROVED** [review]
Looks good to me

**Pull request merged**